TabloidMVC/Views/Category/Index.cshtml

```diff
@@ -0,0 +1,41 @@
+﻿@model IEnumerable<TabloidMVC.Models.Category>
+
+@{
+    ViewData["Title"] = "Index";
+}
+
+<h1>Categories</h1>
```

Comment:
Looks good! The only thing is that if you wrap the entire view in ``<div class="container pt-5"> </div>`` it will change the style to look the same as the Post list.

Reply:
Gonna fix once all tickets are merged